CONTRIBUTING.md

```diff
@@ -1,6 +1,34 @@
 # Contributing
 
-We love pull requests. If you have something you want to add or remove, please open a new pull request.
+If you have something you want to add or remove, please use the following agreed upon process.
+
+### Initial Proposal Creation
+ - Proposals are made by creating a GitHub [issue](https://help.github.com/articles/creating-an-issue/) or [pull request](https://help.github.com/articles/creating-a-pull-request/) (PR).
+ - Issues containing the proposal are tagged with the area(s) under which they fall using [GitHub labels](https://github.com/sul-dlss/DeveloperPlaybook/labels).
+ - The proposal is an appropriate granularity requirement to capture existing consensus.
+ - The proposal is of the form When <doing activity>, we [MUST|SHOULD] <use described approach>.
```

Comment:
In the light of proposals that do not necessarily already capture consensus, the terms 'MUST' and 'SHOULD' imply obligations that are not consistent with the goals of the playbook to document accepted practices, without constraining healthy creativity that may diverge from the current practices (for good reasons).